addon/components/connect.js

```diff
@@ -53,18 +63,14 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
       },
 
       handleChange() {
-        const redux = this.get('redux');
-
-        const props = stateToComputed.call(this, redux.getState(), this.getAttrs());
-
-        const notifyProperties = Object.keys(props).filter(name => {
-          return this.get(name) !== props[name];
-        });
-
-        if (notifyProperties.length > 0) {
-          run.join(() => {
-            notifyProperties.forEach(name => this.notifyPropertyChange(name));
-          });
+        Ember.warn(
```

Comment:
Right on - I think you've done the right thing here moving us forward while warning people of a soon to be dead api :)